src/components/views/ViewDataTable/utils.tsx

```diff
@@ -0,0 +1,63 @@
+import { GridColDef } from '@mui/x-data-grid-pro';
+
+import { COLUMN_TYPE, ZetkinViewColumn } from 'types/views';
+import LocalPersonViewCell, { LocalPersonViewCellParams } from './cells/LocalPersonViewCell';
+import PersonNotesViewCell, { PersonNotesViewCellParams } from './cells/PersonNotesViewCell';
+import SurveyResponseViewCell, { SurveyResponseViewCellParams } from './cells/SurveyResponseViewCell';
+import SurveySubmittedViewCell, { SurveySubmittedViewCellParams } from './cells/SurveySubmittedViewCell';
+
+
+export function colIdFromFieldName(colFieldName : string) : number {
+    // colFieldName is a string like col_10, where 10 is the
+    // ID of the view column that this refers to.
+    const nameFields = colFieldName.split('_');
+    return parseInt(nameFields[1]);
+}
+
+export function makeGridColDef(viewCol: ZetkinViewColumn, orgId: number | string) : GridColDef {
```

Comment:
Would love a comment here that said something like "maps a `ZetkinViewColumn` to an MUI DataGrid `GridColDef` to show the correct content." Just because I love comments.